stdeb.cfg

```diff
@@ -1,5 +1,6 @@
 [rospkg]
 Debian-Version: 100
+Depends: python-catkin-pkg
```

Comment:
`Depends` takes a comma separated list (rather than multiple entries of `Depends`).

Also the same dependency is necessary for Python 3 (`Depends3`).

Reply:
I see. Updated.